Update order-management.pt.md (#1977)
mercadopago/devsite-docs: guides/mp-delivery/order-management.pt.md

### Modalidade de logística Dropoff

> Essa modalidade de logística é utilizada por restaurantes que concordaram que empresas logísticas, que estejam integradas com Mercado Pago, realizem a entrega dos pedidos. Nesse fluxo, logo após aceitar um pedido, será enviada uma notificação de que um entregador estará a caminho para receber o pedido. É importante notar que diferentemente do fluxo que foi falado anteriormente, nessa modalidade de Dropoff os pedidos não terão um código QR, e devido a isso, ao realizar uma consulta sobre o pedido utilizando a API do Mercado Pago Delivery, o atributo "extension.qr" será vazio. **Com esse tipo de logística, é possível que tenhamos um entregador atribuído antes do pedido ter sido aceito. Ao aceitar o pedido seguindo o fluxo normal, seria mostrado o status/substatus: ready_to_ship / ready_to_print ao realizar o primeiro GET. Mas nos casos em que temos um entregador atribuído muito rapidamente (antes de aceitar o pedido) é possível ter o status/substatus: ready_to_ship / on_route_to_pickup no primeiro GET. Portanto, a recomendação para esse tipo de logística é aceitar os pedidos observando apenas o status (ready_to_ship). Mesmo com o entregador atribuído, é necessário aceitar o pedido em até 5 minutos após sua criação, caso contrário, ele será cancelada por Timeout. E uma vez atribuído o entregador, os estados anteriores do fluxo não serão mostrados no GET.** A seguir, segue a descrição dos status de todas as notificações que chegarão ao webhook de pedidos atrelados a essa modalide de logística:

Reviewer: ```suggestion
Essa modalidade de logística é utilizada por restaurantes que concordaram que empresas logísticas, que estejam integradas com Mercado Pago, realizem a entrega dos seus pedidos. Nesse fluxo, logo após aceitar um pedido, será enviada uma notificação de que um entregador estará a caminho para receber o pedido. É importante notar que diferentemente do fluxo que foi apresentado anteriormente, nessa modalidade de Dropoff os pedidos não terão um código QR e, devido a isso, ao realizar uma consulta sobre o pedido utilizando a API do Mercado Pago Delivery, o atributo `extension.qr` será vazio. **Com esse tipo de logística, é possível que tenhamos um entregador atribuído antes do pedido ter sido aceito. Ao aceitar o pedido seguindo o fluxo normal, seria mostrado o status/substatus: `ready_to_ship` / `ready_to_print` ao realizar o primeiro GET, mas nos casos em que há um entregador atribuído muito rapidamente (antes de aceitar o pedido) é possível ter o status/substatus: `ready_to_ship` / `on_route_to_pickup` no primeiro GET. Portanto, a recomendação para esse tipo de logística é aceitar os pedidos observando apenas o status (`ready_to_ship`). Mesmo com o entregador atribuído, é necessário aceitar o pedido em até 5 minutos após sua criação e, caso contrário, ele será cancelada por timeout. E uma vez atribuído o entregador, os estados anteriores do fluxo não serão mostrados no GET.** A seguir, segue a descrição dos status de todas as notificações que chegarão ao Webhook de pedidos atrelados a essa modalide de logística:
```

>   * **ready_to_ship/ready_to_print:** Nesse Status, alguma ação deve ser realizada (aceitar ou cancelar) em no máximo 5 minutos, caso contrário o pedido será cancelado por timeout.

Reviewer: ```suggestion
  * **ready_to_ship/ready_to_print:** Nesse status, alguma ação deve ser realizada (aceitar ou cancelar) em no máximo 5 minutos, caso contrário o pedido será cancelado por timeout.
```

  * **cancelled/cancelled_manually:** Status indicando que uma operação de cancelamento foi executada no pedido.

>   * **cancelled/time_expired:** Nesse Status, o pedido foi cancelado por timeout devido ao fato de não ter sido realizada nenhuma operação nos primeiros 5 minutos desde sua criação.

Reviewer: ```suggestion
  * **cancelled/time_expired:** Nesse status, o pedido foi cancelado por timeout devido ao fato de não ter sido realizada nenhuma operação nos primeiros 5 minutos desde sua criação.
```
